certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -88,6 +92,10 @@
     {
       "name": "Scenarios",
       "description": "Operations on Scenarios"
+    },
+    {
+      "name": "Webhooks",
+      "description": "Webhooks allow to subscribe to notifications about changes in Meisterplan. An HTTP request with a small payload is sent to the specified endpoint when an event that you have subscribed to occurs. Currently webhooks support the following types of events:\n\n-   Project Created - Sends a notification if a project is created in     the subscribed scenario -   Project Updated - Sends a notification if a project is updated or a     sub-entity is created or updated for the subscribed scenario. -   Specific Project Updated - Sends a notification if the project is     updated or a sub-entity is created or updated for the subscribed     project. -   Project Deleted - Sends a notification if a project is deleted for     the subscribed scenario. -   Specific Project Deleted - Sends a notification if the subscribed     project is deleted.\n\nWebhooks Security\n\nMeisterplan optionally supports signing of notification payloads. The process has the following steps:\n\n-   A webhook can be registered with a secret. -   When an event occurs, Meisterplan hashes the notification payload     with this secret using HmacSHA256 algorithm and sends the resulting     signature hex encoded with the X-Hook-Signature header. -   The client can verify the authenticity of the notification by     calculating the HmacSHA256 signature and comparing it to the value     of the X-Hook-Signature header. The signature is calculated using     the secret and the payload.\n\nWebhook Callback Acknowledgement\n\nThe client must respond to an event callback with a 200 HTTP status code to acknowledge that the callback was received. The client should respond immediately after receiving the notification. There is a 10 second timeout. If the client responds with an error (except HTTP status code 410), Meisterplan will retry the notification delivery up to 8 times (delayed using exponential backoff). If too many notifications cannot be delivered, the webhook will be suspended. Suspended webhooks are eventually deleted. Meisterplan webhooks do not follow redirects. You can get more information about the notification schema by calling the /v1/meta/notification/dynamicSchema endpoint."
```

Comment:
@itd-sk Please remove the extra white spaces in the description. please remove from all parameters descriptions in the swagger file

Reply:
@Amjed-Ayoub Removed the white spaces and shortened the description.